"2019-09-25 08:27:13,661 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.FetchSessionHandler : [Consumer clientId=consumer-2, groupId=group_id4] Error sending fetch request (sessionId=165578878, epoch=4043) to node 0: org.apache.kafka.common.errors.DisconnectException.
""2019-09-25 08:27:13,726 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Error connecting to node 172.23.238.225:9092 (id: 0 rack: null)
"java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:482)
	at java.base/sun.nio.ch.Net.connect(Net.java:474)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:233)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:214)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:864)
	at org.apache.kafka.clients.NetworkClient.access$700(NetworkClient.java:64)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:1035)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:508)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:271)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:242)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:233)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.awaitMetadataUpdate(ConsumerNetworkClient.java:161)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.ensureCoordinatorReady(AbstractCoordinator.java:243)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.poll(ConsumerCoordinator.java:314)
	at org.apache.kafka.clients.consumer.KafkaConsumer.updateAssignmentMetadataIfNeeded(KafkaConsumer.java:1218)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1175)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1154)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.pollAndInvoke(KafkaMessageListenerContainer.java:743)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.run(KafkaMessageListenerContainer.java:700)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
"2019-09-25 08:27:13,828 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Error connecting to node 172.23.238.225:9092 (id: 0 rack: null)
"java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:482)
	at java.base/sun.nio.ch.Net.connect(Net.java:474)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:233)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:214)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:864)
	at org.apache.kafka.clients.NetworkClient.access$700(NetworkClient.java:64)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:1035)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:508)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:271)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:242)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:233)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.awaitMetadataUpdate(ConsumerNetworkClient.java:161)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.ensureCoordinatorReady(AbstractCoordinator.java:243)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.poll(ConsumerCoordinator.java:314)
	at org.apache.kafka.clients.consumer.KafkaConsumer.updateAssignmentMetadataIfNeeded(KafkaConsumer.java:1218)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1175)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1154)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.pollAndInvoke(KafkaMessageListenerContainer.java:743)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.run(KafkaMessageListenerContainer.java:700)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
"2019-09-25 08:27:14,032 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Error connecting to node 172.23.238.225:9092 (id: 0 rack: null)
"java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:482)
	at java.base/sun.nio.ch.Net.connect(Net.java:474)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:233)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:214)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:864)
	at org.apache.kafka.clients.NetworkClient.access$700(NetworkClient.java:64)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:1035)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:508)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:271)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:242)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:233)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.awaitMetadataUpdate(ConsumerNetworkClient.java:161)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.ensureCoordinatorReady(AbstractCoordinator.java:243)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.poll(ConsumerCoordinator.java:314)
	at org.apache.kafka.clients.consumer.KafkaConsumer.updateAssignmentMetadataIfNeeded(KafkaConsumer.java:1218)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1175)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1154)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.pollAndInvoke(KafkaMessageListenerContainer.java:743)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.run(KafkaMessageListenerContainer.java:700)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
"2019-09-25 08:27:14,387 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Error connecting to node 172.23.238.225:9092 (id: 0 rack: null)
"java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:482)
	at java.base/sun.nio.ch.Net.connect(Net.java:474)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:233)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:214)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:864)
	at org.apache.kafka.clients.NetworkClient.access$700(NetworkClient.java:64)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:1035)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:508)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:271)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:242)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:233)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.awaitMetadataUpdate(ConsumerNetworkClient.java:161)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.ensureCoordinatorReady(AbstractCoordinator.java:243)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.poll(ConsumerCoordinator.java:314)
	at org.apache.kafka.clients.consumer.KafkaConsumer.updateAssignmentMetadataIfNeeded(KafkaConsumer.java:1218)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1175)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1154)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.pollAndInvoke(KafkaMessageListenerContainer.java:743)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.run(KafkaMessageListenerContainer.java:700)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
"2019-09-25 08:27:15,193 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Error connecting to node 172.23.238.225:9092 (id: 0 rack: null)
"java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:482)
	at java.base/sun.nio.ch.Net.connect(Net.java:474)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:233)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:214)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:864)
	at org.apache.kafka.clients.NetworkClient.access$700(NetworkClient.java:64)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:1035)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:508)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:271)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:242)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:233)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.awaitMetadataUpdate(ConsumerNetworkClient.java:161)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.ensureCoordinatorReady(AbstractCoordinator.java:243)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.poll(ConsumerCoordinator.java:314)
	at org.apache.kafka.clients.consumer.KafkaConsumer.updateAssignmentMetadataIfNeeded(KafkaConsumer.java:1218)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1175)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1154)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.pollAndInvoke(KafkaMessageListenerContainer.java:743)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.run(KafkaMessageListenerContainer.java:700)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
"2019-09-25 08:27:16,399 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Error connecting to node 172.23.238.225:9092 (id: 0 rack: null)
"java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:482)
	at java.base/sun.nio.ch.Net.connect(Net.java:474)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:233)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:214)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:864)
	at org.apache.kafka.clients.NetworkClient.access$700(NetworkClient.java:64)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:1035)
	at org.apache.kafka.clients.NetworkClient$DefaultMetadataUpdater.maybeUpdate(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.poll(NetworkClient.java:508)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:271)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:242)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.poll(ConsumerNetworkClient.java:233)
	at org.apache.kafka.clients.consumer.internals.ConsumerNetworkClient.awaitMetadataUpdate(ConsumerNetworkClient.java:161)
	at org.apache.kafka.clients.consumer.internals.AbstractCoordinator.ensureCoordinatorReady(AbstractCoordinator.java:243)
	at org.apache.kafka.clients.consumer.internals.ConsumerCoordinator.poll(ConsumerCoordinator.java:314)
	at org.apache.kafka.clients.consumer.KafkaConsumer.updateAssignmentMetadataIfNeeded(KafkaConsumer.java:1218)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1175)
	at org.apache.kafka.clients.consumer.KafkaConsumer.poll(KafkaConsumer.java:1154)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.pollAndInvoke(KafkaMessageListenerContainer.java:743)
	at org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer.run(KafkaMessageListenerContainer.java:700)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
"2019-09-25 08:27:17,574 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Discovered group coordinator 172.23.238.225:9092 (id: 2147483647 rack: null)
""2019-09-25 08:27:17,583 ERROR [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Offset commit failed on partition kafka-Example4-0 at offset 28: The coordinator is not aware of this member.
""2019-09-25 08:27:17,584 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Asynchronous auto-commit of offsets {kafka-Example4-0=OffsetAndMetadata{offset=28, metadata=''}} failed: Commit cannot be completed since the group has already rebalanced and assigned the partitions to another member. This means that the time between subsequent calls to poll() was longer than the configured max.poll.interval.ms, which typically implies that the poll loop is spending too much time message processing. You can address this either by increasing the session timeout or by reducing the maximum size of batches returned in poll() with max.poll.records.
""2019-09-25 08:27:17,584 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Synchronous auto-commit of offsets {kafka-Example4-0=OffsetAndMetadata{offset=28, metadata=''}} failed: Commit cannot be completed since the group has already rebalanced and assigned the partitions to another member. This means that the time between subsequent calls to poll() was longer than the configured max.poll.interval.ms, which typically implies that the poll loop is spending too much time message processing. You can address this either by increasing the session timeout or by reducing the maximum size of batches returned in poll() with max.poll.records.
""2019-09-25 08:27:17,584 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Revoking previously assigned partitions [kafka-Example4-0]
""2019-09-25 08:27:17,584 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.kafka.listener.KafkaMessageListenerContainer : partitions revoked: [kafka-Example4-0]
""2019-09-25 08:27:17,584 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] (Re-)joining group
""2019-09-25 08:27:17,597 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Successfully joined group with generation 15
""2019-09-25 08:27:17,598 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Setting newly assigned partitions [kafka-Example4-0]
""2019-09-25 08:27:17,601 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.kafka.listener.KafkaMessageListenerContainer : partitions assigned: [kafka-Example4-0]
""2019-09-25 09:05:43,930 INFO  [kafka-coordinator-heartbeat-thread | group_id4] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Group coordinator 172.23.238.225:9092 (id: 2147483647 rack: null) is unavailable or invalid, will attempt rediscovery
""2019-09-25 09:06:00,828 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Discovered group coordinator 172.23.238.225:9092 (id: 2147483647 rack: null)
""2019-09-25 09:36:31,788 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler : Shutting down ExecutorService
""2019-09-25 09:36:31,892 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer : Consumer stopped
""2019-09-25 09:36:31,896 INFO  [Thread-8] org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor : Shutting down ExecutorService 'applicationTaskExecutor'
""2019-09-25 09:36:31,901 INFO  [Thread-8] org.mongodb.driver.connection : Closed connection [connectionId{localValue:2, serverValue:2}] to localhost:27017 because the pool has been closed.
""2019-09-25 10:01:35,036 INFO  [restartedMain] com.stackroute.Application : Starting Application on talla-ThinkPad-T450 with PID 22421 (/home/talla/Downloads/searchMicroservice/target/classes started by talla in /home/talla/Downloads/searchMicroservice)
""2019-09-25 10:01:35,065 INFO  [restartedMain] com.stackroute.Application : No active profile set, falling back to default profiles: default
""2019-09-25 10:01:35,511 INFO  [restartedMain] org.springframework.boot.devtools.env.DevToolsPropertyDefaultsPostProcessor : Devtools property defaults active! Set 'spring.devtools.add-properties' to 'false' to disable
""2019-09-25 10:01:35,512 INFO  [restartedMain] org.springframework.boot.devtools.env.DevToolsPropertyDefaultsPostProcessor : For additional web related logging consider setting the 'logging.level.web' property to 'DEBUG'
""2019-09-25 10:01:42,519 INFO  [restartedMain] org.springframework.data.repository.config.RepositoryConfigurationDelegate : Bootstrapping Spring Data repositories in DEFAULT mode.
""2019-09-25 10:01:43,058 INFO  [restartedMain] org.springframework.data.repository.config.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 516ms. Found 1 repository interfaces.
""2019-09-25 10:01:45,181 INFO  [restartedMain] org.springframework.context.support.PostProcessorRegistrationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.kafka.annotation.KafkaBootstrapConfiguration' of type [org.springframework.kafka.annotation.KafkaBootstrapConfiguration$$EnhancerBySpringCGLIB$$ffe7f626] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
""2019-09-25 10:01:47,646 INFO  [restartedMain] org.springframework.boot.web.embedded.tomcat.TomcatWebServer : Tomcat initialized with port(s): 8070 (http)
""2019-09-25 10:01:47,993 INFO  [restartedMain] org.apache.catalina.core.StandardService : Starting service [Tomcat]
""2019-09-25 10:01:48,008 INFO  [restartedMain] org.apache.catalina.core.StandardEngine : Starting Servlet engine: [Apache Tomcat/9.0.21]
""2019-09-25 10:01:48,789 INFO  [restartedMain] org.apache.catalina.core.ContainerBase.[Tomcat].[localhost].[/] : Initializing Spring embedded WebApplicationContext
""2019-09-25 10:01:51,484 INFO  [restartedMain] org.mongodb.driver.cluster : Cluster created with settings {hosts=[localhost:27017], mode=MULTIPLE, requiredClusterType=UNKNOWN, serverSelectionTimeout='30000 ms', maxWaitQueueSize=500}
""2019-09-25 10:01:51,498 INFO  [restartedMain] org.mongodb.driver.cluster : Adding discovered server localhost:27017 to client view of cluster
""2019-09-25 10:01:51,882 INFO  [cluster-ClusterId{value='5d8aedb70d345857956e5907', description='null'}-localhost:27017] org.mongodb.driver.connection : Opened connection [connectionId{localValue:1, serverValue:3}] to localhost:27017
""2019-09-25 10:01:51,919 INFO  [cluster-ClusterId{value='5d8aedb70d345857956e5907', description='null'}-localhost:27017] org.mongodb.driver.cluster : Monitor thread successfully connected to server with description ServerDescription{address=localhost:27017, type=STANDALONE, state=CONNECTED, ok=true, version=ServerVersion{versionList=[3, 6, 3]}, minWireVersion=0, maxWireVersion=6, maxDocumentSize=16777216, logicalSessionTimeoutMinutes=30, roundTripTimeNanos=5538563}
""2019-09-25 10:01:51,926 INFO  [cluster-ClusterId{value='5d8aedb70d345857956e5907', description='null'}-localhost:27017] org.mongodb.driver.cluster : Discovered cluster type of STANDALONE
""2019-09-25 10:01:53,020 INFO  [restartedMain] org.springframework.boot.devtools.autoconfigure.OptionalLiveReloadServer : LiveReload server is running on port 35729
""2019-09-25 10:01:57,129 INFO  [restartedMain] org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor : Initializing ExecutorService 'applicationTaskExecutor'
""2019-09-25 10:01:58,844 INFO  [restartedMain] org.apache.kafka.clients.consumer.ConsumerConfig : ConsumerConfig values: 
	auto.commit.interval.ms = 5000
	auto.offset.reset = latest
	bootstrap.servers = [172.23.238.225:9092]
	check.crcs = true
	client.id = 
	connections.max.idle.ms = 540000
	default.api.timeout.ms = 60000
	enable.auto.commit = true
	exclude.internal.topics = true
	fetch.max.bytes = 52428800
	fetch.max.wait.ms = 500
	fetch.min.bytes = 1
	group.id = group_id4
	heartbeat.interval.ms = 3000
	interceptor.classes = []
	internal.leave.group.on.close = true
	isolation.level = read_uncommitted
	key.deserializer = class org.apache.kafka.common.serialization.StringDeserializer
	max.partition.fetch.bytes = 1048576
	max.poll.interval.ms = 300000
	max.poll.records = 500
	metadata.max.age.ms = 300000
	metric.reporters = []
	metrics.num.samples = 2
	metrics.recording.level = INFO
	metrics.sample.window.ms = 30000
	partition.assignment.strategy = [class org.apache.kafka.clients.consumer.RangeAssignor]
	receive.buffer.bytes = 65536
	reconnect.backoff.max.ms = 1000
	reconnect.backoff.ms = 50
	request.timeout.ms = 30000
	retry.backoff.ms = 100
	sasl.client.callback.handler.class = null
	sasl.jaas.config = null
	sasl.kerberos.kinit.cmd = /usr/bin/kinit
	sasl.kerberos.min.time.before.relogin = 60000
	sasl.kerberos.service.name = null
	sasl.kerberos.ticket.renew.jitter = 0.05
	sasl.kerberos.ticket.renew.window.factor = 0.8
	sasl.login.callback.handler.class = null
	sasl.login.class = null
	sasl.login.refresh.buffer.seconds = 300
	sasl.login.refresh.min.period.seconds = 60
	sasl.login.refresh.window.factor = 0.8
	sasl.login.refresh.window.jitter = 0.05
	sasl.mechanism = GSSAPI
	security.protocol = PLAINTEXT
	send.buffer.bytes = 131072
	session.timeout.ms = 10000
	ssl.cipher.suites = null
	ssl.enabled.protocols = [TLSv1.2, TLSv1.1, TLSv1]
	ssl.endpoint.identification.algorithm = https
	ssl.key.password = null
	ssl.keymanager.algorithm = SunX509
	ssl.keystore.location = null
	ssl.keystore.password = null
	ssl.keystore.type = JKS
	ssl.protocol = TLS
	ssl.provider = null
	ssl.secure.random.implementation = null
	ssl.trustmanager.algorithm = PKIX
	ssl.truststore.location = null
	ssl.truststore.password = null
	ssl.truststore.type = JKS
	value.deserializer = class org.springframework.kafka.support.serializer.JsonDeserializer

""2019-09-25 10:01:59,297 WARN  [restartedMain] org.apache.kafka.clients.consumer.ConsumerConfig : The configuration 'spring.json.trusted.packages' was supplied but isn't a known config.
""2019-09-25 10:01:59,306 INFO  [restartedMain] org.apache.kafka.common.utils.AppInfoParser : Kafka version : 2.0.1
""2019-09-25 10:01:59,309 INFO  [restartedMain] org.apache.kafka.common.utils.AppInfoParser : Kafka commitId : fa14705e51bd2ce5
""2019-09-25 10:01:59,857 INFO  [restartedMain] org.apache.kafka.clients.Metadata : Cluster ID: S9W8bfzwSzKRphGLM3xSfw
""2019-09-25 10:01:59,954 INFO  [restartedMain] org.apache.kafka.clients.consumer.ConsumerConfig : ConsumerConfig values: 
	auto.commit.interval.ms = 5000
	auto.offset.reset = latest
	bootstrap.servers = [172.23.238.225:9092]
	check.crcs = true
	client.id = 
	connections.max.idle.ms = 540000
	default.api.timeout.ms = 60000
	enable.auto.commit = true
	exclude.internal.topics = true
	fetch.max.bytes = 52428800
	fetch.max.wait.ms = 500
	fetch.min.bytes = 1
	group.id = group_id4
	heartbeat.interval.ms = 3000
	interceptor.classes = []
	internal.leave.group.on.close = true
	isolation.level = read_uncommitted
	key.deserializer = class org.apache.kafka.common.serialization.StringDeserializer
	max.partition.fetch.bytes = 1048576
	max.poll.interval.ms = 300000
	max.poll.records = 500
	metadata.max.age.ms = 300000
	metric.reporters = []
	metrics.num.samples = 2
	metrics.recording.level = INFO
	metrics.sample.window.ms = 30000
	partition.assignment.strategy = [class org.apache.kafka.clients.consumer.RangeAssignor]
	receive.buffer.bytes = 65536
	reconnect.backoff.max.ms = 1000
	reconnect.backoff.ms = 50
	request.timeout.ms = 30000
	retry.backoff.ms = 100
	sasl.client.callback.handler.class = null
	sasl.jaas.config = null
	sasl.kerberos.kinit.cmd = /usr/bin/kinit
	sasl.kerberos.min.time.before.relogin = 60000
	sasl.kerberos.service.name = null
	sasl.kerberos.ticket.renew.jitter = 0.05
	sasl.kerberos.ticket.renew.window.factor = 0.8
	sasl.login.callback.handler.class = null
	sasl.login.class = null
	sasl.login.refresh.buffer.seconds = 300
	sasl.login.refresh.min.period.seconds = 60
	sasl.login.refresh.window.factor = 0.8
	sasl.login.refresh.window.jitter = 0.05
	sasl.mechanism = GSSAPI
	security.protocol = PLAINTEXT
	send.buffer.bytes = 131072
	session.timeout.ms = 10000
	ssl.cipher.suites = null
	ssl.enabled.protocols = [TLSv1.2, TLSv1.1, TLSv1]
	ssl.endpoint.identification.algorithm = https
	ssl.key.password = null
	ssl.keymanager.algorithm = SunX509
	ssl.keystore.location = null
	ssl.keystore.password = null
	ssl.keystore.type = JKS
	ssl.protocol = TLS
	ssl.provider = null
	ssl.secure.random.implementation = null
	ssl.trustmanager.algorithm = PKIX
	ssl.truststore.location = null
	ssl.truststore.password = null
	ssl.truststore.type = JKS
	value.deserializer = class org.springframework.kafka.support.serializer.JsonDeserializer

""2019-09-25 10:01:59,979 WARN  [restartedMain] org.apache.kafka.clients.consumer.ConsumerConfig : The configuration 'spring.json.trusted.packages' was supplied but isn't a known config.
""2019-09-25 10:01:59,979 INFO  [restartedMain] org.apache.kafka.common.utils.AppInfoParser : Kafka version : 2.0.1
""2019-09-25 10:01:59,979 INFO  [restartedMain] org.apache.kafka.common.utils.AppInfoParser : Kafka commitId : fa14705e51bd2ce5
""2019-09-25 10:02:00,000 INFO  [restartedMain] org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler : Initializing ExecutorService
""2019-09-25 10:02:00,159 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.Metadata : Cluster ID: S9W8bfzwSzKRphGLM3xSfw
""2019-09-25 10:02:00,160 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Discovered group coordinator 172.23.238.225:9092 (id: 2147483647 rack: null)
""2019-09-25 10:02:00,214 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Revoking previously assigned partitions []
""2019-09-25 10:02:00,214 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.kafka.listener.KafkaMessageListenerContainer : partitions revoked: []
""2019-09-25 10:02:00,214 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] (Re-)joining group
""2019-09-25 10:02:00,277 INFO  [restartedMain] org.springframework.boot.web.embedded.tomcat.TomcatWebServer : Tomcat started on port(s): 8070 (http) with context path ''
""2019-09-25 10:02:00,295 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Successfully joined group with generation 17
""2019-09-25 10:02:00,306 INFO  [restartedMain] com.stackroute.Application : Started Application in 29.755 seconds (JVM running for 34.394)
""2019-09-25 10:02:00,310 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Setting newly assigned partitions [kafka-Example4-0]
""2019-09-25 10:02:00,322 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.kafka.listener.KafkaMessageListenerContainer : partitions assigned: [kafka-Example4-0]
""2019-09-25 11:35:20,725 INFO  [kafka-coordinator-heartbeat-thread | group_id4] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Group coordinator 172.23.238.225:9092 (id: 2147483647 rack: null) is unavailable or invalid, will attempt rediscovery
""2019-09-25 11:35:43,740 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.FetchSessionHandler : [Consumer clientId=consumer-2, groupId=group_id4] Error sending fetch request (sessionId=1898336602, epoch=10855) to node 0: org.apache.kafka.common.errors.DisconnectException.
""2019-09-25 11:37:54,422 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:40:05,493 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:12,053 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:15,481 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:18,549 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:21,621 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:24,693 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:27,765 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:30,837 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:33,909 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:36,981 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:40,053 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:43,125 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:41:45,198 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Discovered group coordinator 172.23.238.225:9092 (id: 2147483647 rack: null)
""2019-09-25 11:41:45,209 ERROR [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Offset commit failed on partition kafka-Example4-0 at offset 28: The coordinator is not aware of this member.
""2019-09-25 11:41:45,210 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Asynchronous auto-commit of offsets {kafka-Example4-0=OffsetAndMetadata{offset=28, metadata=''}} failed: Commit cannot be completed since the group has already rebalanced and assigned the partitions to another member. This means that the time between subsequent calls to poll() was longer than the configured max.poll.interval.ms, which typically implies that the poll loop is spending too much time message processing. You can address this either by increasing the session timeout or by reducing the maximum size of batches returned in poll() with max.poll.records.
""2019-09-25 11:41:45,211 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Synchronous auto-commit of offsets {kafka-Example4-0=OffsetAndMetadata{offset=28, metadata=''}} failed: Commit cannot be completed since the group has already rebalanced and assigned the partitions to another member. This means that the time between subsequent calls to poll() was longer than the configured max.poll.interval.ms, which typically implies that the poll loop is spending too much time message processing. You can address this either by increasing the session timeout or by reducing the maximum size of batches returned in poll() with max.poll.records.
""2019-09-25 11:41:45,212 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Revoking previously assigned partitions [kafka-Example4-0]
""2019-09-25 11:41:45,212 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.kafka.listener.KafkaMessageListenerContainer : partitions revoked: [kafka-Example4-0]
""2019-09-25 11:41:45,213 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] (Re-)joining group
""2019-09-25 11:41:45,222 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Successfully joined group with generation 19
""2019-09-25 11:41:45,223 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.consumer.internals.ConsumerCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Setting newly assigned partitions [kafka-Example4-0]
""2019-09-25 11:41:45,226 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.kafka.listener.KafkaMessageListenerContainer : partitions assigned: [kafka-Example4-0]
""2019-09-25 11:50:45,275 INFO  [kafka-coordinator-heartbeat-thread | group_id4] org.apache.kafka.clients.consumer.internals.AbstractCoordinator : [Consumer clientId=consumer-2, groupId=group_id4] Group coordinator 172.23.238.225:9092 (id: 2147483647 rack: null) is unavailable or invalid, will attempt rediscovery
""2019-09-25 11:51:06,141 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.FetchSessionHandler : [Consumer clientId=consumer-2, groupId=group_id4] Error sending fetch request (sessionId=1459638084, epoch=1005) to node 0: org.apache.kafka.common.errors.DisconnectException.
""2019-09-25 11:53:16,025 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:55:27,092 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:56:35,701 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:56:39,125 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:56:42,197 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:56:45,269 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:56:48,341 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:56:51,413 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:56:54,484 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:56:57,557 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:00,629 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:03,701 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:06,773 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:09,845 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:12,917 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:15,989 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:19,061 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:22,133 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:25,205 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:28,277 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:31,349 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:34,420 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:37,493 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:40,565 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:43,637 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:46,709 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:49,781 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:52,853 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:57:55,925 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:58:14,388 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:58:18,292 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:58:21,364 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:58:24,437 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:58:27,508 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:58:30,581 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 11:58:33,653 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 14:47:31,353 WARN  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.apache.kafka.clients.NetworkClient : [Consumer clientId=consumer-2, groupId=group_id4] Connection to node 0 could not be established. Broker may not be available.
""2019-09-25 14:47:42,051 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler : Shutting down ExecutorService
""2019-09-25 14:47:42,055 INFO  [org.springframework.kafka.KafkaListenerEndpointContainer#0-0-C-1] org.springframework.kafka.listener.KafkaMessageListenerContainer$ListenerConsumer : Consumer stopped
""2019-09-25 14:47:42,062 INFO  [Thread-8] org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor : Shutting down ExecutorService 'applicationTaskExecutor'
"